core/src/main/java/org/apache/carbondata/core/datastore/page/UnsafeVarLengthColumnPage.java

```diff
@@ -42,6 +42,16 @@
     baseOffset = memoryBlock.getBaseOffset();
   }
 
+  UnsafeVarLengthColumnPage(TableSpec.ColumnSpec columnSpec, DataType dataType, int pageSize,
```

Comment:
It is not used anywhere, please remove it if not required

Reply:
fixed